modules/040-node-manager/templates/node-group/_bootstrap.tpl

```diff
@@ -41,13 +41,19 @@ import json
 import ssl
 
 try:
-    from urllib.request import urlopen, Request
+    from urllib.request import urlopen, Request, HTTPError
 except ImportError as e:
-    from urllib2 import urlopen, Request
+    from urllib2 import urlopen, Request, HTTPError
 
 ssl.match_hostname = lambda cert, hostname: True
 request = Request(sys.argv[1], headers={'Authorization': 'Bearer ' + sys.argv[2]})
-response = urlopen(request, cafile='/var/lib/bashible/ca.crt')
+try:
+    response = urlopen(request, cafile='/var/lib/bashible/ca.crt')
+except HTTPError as e:
+    if e.getcode() >= 401:
```

Comment:
We need to handle only 401 and 403 error code

Reply:
Done